lib/Sema/CSSimplify.cpp

```diff
@@ -8818,6 +8818,13 @@ ConstraintSystem::simplifyPackElementOfConstraint(Type first, Type second,
     return SolutionKind::Solved;
   }
 
+  if (shouldAttemptFixes()) {
+    auto *loc = getConstraintLocator(locator);
+    auto *fix = AllowInvalidPackElement::create(*this, packType, loc);
+    if (!recordFix(fix))
```

Comment:
Should this has a higher impact than one?

Reply:
At least right now, packs can only be direct references to parameters or local variables, so we can't get into any overload situations just yet